cmd/wksctl/profile/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+package constants
+
+const (
+	ProfilesStorePrefix = "profiles"
```

Comment:
If this means the directory name that's expected to be in the user's CWD, then it would feel better to make this value a flag with a default value of `profiles` - so that we allow the user to use the command from within another directory.

Reply:
thank you fixed per commit